src/FrontendAccountCreation.Web/Views/Organisation/AddApprovedPerson.cshtml

```diff
@@ -0,0 +1,110 @@
+﻿@using FrontendAccountCreation.Core.Sessions;
+@using FrontendAccountCreation.Web.Extensions
+@using FrontendAccountCreation.Web.ViewModels.ReExAccount
+@using FrontendAccountCreation.Web.ViewModels.Shared.GovUK;
+
+@model AddApprovedPersonViewModel;
+
+@{
+    ViewData["Title"] = Localizer["AddAnApprovedPerson.Title"];
+    var errorsViewModel = new ErrorsViewModel(ViewData.ModelState.ToErrorDictionary(), Localizer);
+}
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper govuk-!-padding-top-4" id="main-content" role="main">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+
+                @await Html.PartialAsync("Partials/Govuk/_ErrorSummary", errorsViewModel)
+
+                <form asp-controller="AccountCreation" method="post">
+                    <div class="govuk-form-group@(!ViewData.ModelState.IsValid ? " govuk-form-group--error" : string.Empty)">
+                        <h1 class="govuk-heading-l govuk-!-margin-bottom-4">
+                            @Localizer["AddAnApprovedPerson.Title"]
+                        </h1>
+
+                        <p class="govuk-body">You are eligible to become an 'approved person' or you can invite somebody else to do this. An approved person needs to be invited to take on tasks on behalf of the organisation.</p>
```

Comment:
This text also need to go as part of resource we need to support Welsh translation. Basically all content should have welsh translation